test/RunSpec.hs

```diff
@@ -97,6 +97,45 @@ spec = do
         rmDir "test/integration/custom-package-conf/packages/"
         rmDir "test/integration/custom-package-conf/foo/dist/"
 
+    it "prints verbose description of a specification" $ do
+      (r, ()) <- hCapture [stderr] $ doctest ["--verbose", "test/integration/testSimple/Fib.hs"]
+      r `shouldBe` unlines [
+          "### Started execution at test/integration/testSimple/Fib.hs:5."
+        , "### HSpec specification:"
+        , "fib 10"
+        , "### Successful!"
+        , ""
+        , "# Final summary:"
+        , "Examples: 1  Tried: 1  Errors: 0  Failures: 0"
+        ]
+
+    it "prints verbose description of a property" $ do
+      (r, ()) <- hCapture [stderr] $ doctest ["--verbose", "test/integration/property-bool/Foo.hs"]
+      r `shouldBe` unlines [
+          "### Started execution at test/integration/property-bool/Foo.hs:4."
+        , "### QuickCheck property:"
+        , "True"
+        , "### Successful!"
+        , ""
+        , "# Final summary:"
+        , "Examples: 1  Tried: 1  Errors: 0  Failures: 0"
+        ]
+
+    it "prints verbose error" $ do
+      (r, e) <- hCapture [stderr] . E.try $ doctest ["--verbose", "test/integration/failing/Foo.hs"]
+      e `shouldBe` Left (ExitFailure 1)
+      r `shouldBe` unlines [
+              "### Started execution at test/integration/failing/Foo.hs:5."
+            , "### HSpec specification:"
```

Comment:
It's not "HSpec" but "Hspec" (for historic reasons).  But here this doesn't really relate to Hspec at all, it's just an example, so maybe something like:

```
### code example:
```

would be fitting.

Reply:
I just could not come up with the right caption. "Example" and "property" fit very well indeed.